Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:3-alpine
```

Comment:
It looks like we do need to force Python version 3.7.9 here.  So, try this instead:

FROM python:3.7.9-alpine